src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
I am expecting to see the radiation from the frame to the zone. The IR from the parent zone is used to calculate the surface temperatures of the frame (though I don't think the IR from the frame is used in the overall IR exchange between other zone surfaces--this is a separate problem, but I imagine it has a relatively small impact).

Because the IR from the parent zone is used to calculate the surface temperatures, the convection to the zone is not equal to the conduction through the frame.